packages/toolpad-app/src/runtime/SignInPage.tsx

```diff
@@ -16,14 +16,14 @@ export default function SignInPage() {
   const { signIn, isSigningIn } = React.useContext(AuthContext);
 
   const [errorSnackbarMessage, setErrorSnackbarMessage] = React.useState<string>('');
-  const [latestSelectedProvider, setLatestSelectedProvider] = React.useState<AuthProvider | null>(
-    null,
-  );
+  const [latestSelectedProvider, setLatestSelectedProvider] = React.useState<
+    AuthProvider['provider'] | null
```

Comment:
```
type AuthPrivider = 'Google' | 'GitHub' // just use the same names as in auth.js

interface AuthProviderConfig {
  provider: AuthProvider
}
```

Reply:
Yeah that could be better, should be good now: https://github.com/mui/mui-toolpad/pull/2925/commits/9bbfeec826c153c0976d1af949dea345651de940